test/sqs-e2e/testdata/aws-sqs-queue.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: simple-pipeline
+spec:
+  vertices:
+    - name: in
+      source:
+        generator:
+          rpu: 5
+          duration: 5s
+    - name: cat
+      udf:
+        builtin:
+          name: cat
+    - name: out
+      sink:
+        log: {}
```

Comment:
nit: we can remove `cat` vertex and `out` vertex to better isolate the testing target.